helix-rest/src/test/java/org/apache/helix/rest/server/TestInstancesAccessor.java

```diff
@@ -46,7 +46,7 @@
   private final static String CLUSTER_NAME = "TestCluster_0";
 
   @Test
-  public void testInstancesStoppable_zoneBased() throws IOException {
+  public void testInstancesStoppable_zoneBased() throws Exception {
```

Comment:
nit, No need to change this file with your new modification?

Reply:
Done.